From: Sophie Brun <sophie@offensive-security.com>
Date: Mon, 19 Aug 2019 09:08:03 +0200
Subject: Change configuration file

Last-Update: 2017-11-23

By default MANIP=webymsg is enabled but it fails with "Manipulator webymsg error
Dispatch to manipulator initialization error".
Last-Update: 2017-11-23
---
 config/xplico_cli_fix.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/xplico_cli_fix.cfg b/config/xplico_cli_fix.cfg
index 09c0773..cdfffac 100644
--- a/config/xplico_cli_fix.cfg
+++ b/config/xplico_cli_fix.cfg
@@ -120,7 +120,7 @@ MANIP=webmail MPBIN=/opt/xplico/bin/mwmail
 MANIP=fbwchat MPBIN=/opt/xplico/bin/mfbc
 MANIP=httpfd MPBIN=/opt/xplico/bin/mfile
 MANIP=paltalk_exp MPBIN=/opt/xplico/bin/mpaltalk
-MANIP=webymsg MPBIN=/opt/xplico/bin/mwebymsg
+#MANIP=webymsg MPBIN=/opt/xplico/bin/mwebymsg
 
 
 #---------- TMP and log directories paths
